api/v1alpha1/trench_types.go

```diff
@@ -30,8 +31,6 @@ type TrenchSpec struct {
 
 	// ConfigMapName
 	ConfigMapName string `json:"configMapName,omitempty"`
```

Comment:
Maybe this attribute should be removed since it is an internal Meridio resource and it should be hidden for the users.

Reply:
Agree. I plan to modify this trench CRD in another pull request, maybe also adding some more attributes. But I will leave it as it is, for this pull request is mainly about the vip